lametro/models/cms.py

```diff
@@ -272,7 +272,27 @@ def clean(self):
 class Tooltip(models.Model):
     include_in_dump = True
 
-    target = models.CharField(max_length=255, unique=True)
+    TARGET_CHOICES = [
+        ("board_report_type", "Board Report Type"),
+        ("fiscal_year", "Fiscal Year"),
+        ("geographic_administrative_location", "Geographic / Administrative Location"),
```

Comment:
I went with defining a separate set of similar choices here mostly because of the 3 extra options, and because we probably don't need the `_exact` suffix that the Subject model's choices have